### PR TITLE
Feat: Webapp Commit Message Support

### LINKED ISSUE #590: Upgrade Conventional Commits CI to support GitHub webapp commit messages
I guess we need to upgrade our conventional commits CI to support GitHub webapp commit messages.

_Originally posted by @pavlovcik in https://github.com/ubiquity/ubiquity-dollar/issues/586#issuecomment-1450904927_

### PR DESCRIPTION
**Pull request by Steveantor**
Resolves #590

### CODE REVIEW

**Comment by 0x4007** [comment]
Cool solution. Can you open up a pull request on your own fork and demonstrate that it works and link the proof here?

**Comment by Steveantor** [comment]
> Cool solution. Can you open up a pull request on your own fork and demonstrate that it works and link the proof here?

https://github.com/Steveantor/msgtester/commit/f2ba1313040fb801c635d2d00a6e67474239c104 (Pass)
https://github.com/Steveantor/msgtester/commit/488cc18867300d3b958eb5692190100df0819ced (Fail)
Yes it does @pavlovcik

File: .github/workflows/conventional-commits.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: webiny/action-conventional-commits@v1.1.0
+      - uses: ubiquity/action-conventional-commits@v1.1.2
```
Comment (0x4007) [inline .github/workflows/conventional-commits.yml]:
```suggestion
      - uses: ubiquity/action-conventional-commits@v1.1.2 
```

**Review by 0x4007 — CHANGES_REQUESTED** [review]
When I press "Commit suggestion" the commit message is `Update .github/workflows/conventional-commits.yml` which doesn't exactly match the format that you used in your examples (you prefixed `Update: `) 

Can you check again the GitHub suggestion commit name syntax and verify that it is supported?

image

File: .github/workflows/conventional-commits.yml [hunk 2]
```diff
@@ -11,4 +11,4 @@ jobs:
...
```
Comment (pavlovcik) [inline .github/workflows/conventional-commits.yml:14]:
```suggestion
      - uses: ubiquity/action-conventional-commits@v1.1.2
```

**Comment by Steveantor** [comment]
> When I press "Commit suggestion" the commit message is `Update .github/workflows/conventional-commits.yml` which doesn't exactly match the format that you used in your examples (you prefixed `Update: `)
> 
> Can you check again the GitHub suggestion commit name syntax and verify that it is supported?

I believe it does

**Comment by 0x4007** [comment]
> I believe it does

I guess the best way would be if you added me as a contributor on your test repo and I can test by adding a suggestion on your pull request and seeing if it passes. Otherwise I'm skeptical because it doesn't appear that you went through the expected flow of using the suggestion feature and committing directly from the GitHub UI (because your commit message syntax is different)

**Comment by Steveantor** [comment]
> > I believe it does
> 
> I guess the best way would be if you added me as a contributor on your test repo and I can test by adding a suggestion on your pull request and seeing if it passes. Otherwise I'm skeptical because it doesn't appear that you went through the expected flow of using the suggestion feature and committing directly from the GitHub UI (because your commit message syntax is different)

I've invited you now @pavlovcik

**Comment by 0x4007** [comment]
Can you open up a test PR there and I can try the entire flow to verify?

**Comment by Steveantor** [comment]
> > I believe it does
> 
> I guess the best way would be if you added me as a contributor on your test repo and I can test by adding a suggestion on your pull request and seeing if it passes. Otherwise I'm skeptical because it doesn't appear that you went through the expected flow of using the suggestion feature and committing directly from the GitHub UI (because your commit message syntax is different)

done @pavlovcik

**Comment by Steveantor** [comment]
> Can you open up a test PR there and I can try the entire flow to verify?

done @pavlovcik

**Comment by 0x4007** [comment]
Proof: https://github.com/Steveantor/msgtester/pull/1/commits/bc2af285c06d1af1ac5d648c042e00099ee9fb14

**Pull request merged**